frontend/jupyter/src/app/services/actions.service.ts

```diff
@@ -143,7 +143,13 @@ export class ActionsService {
 
             const object = `${namespace}/${name}`;
             const message = $localize`Delete request was sent.`;
-            this.snackBar.open(`${object}: ${message}`, SnackType.Info, 3000);
+            const config: SnackBarConfig = {
+              data: {
+                msg: `${object}: ${message}`,
+                snackType: SnackType.Info,
+              },
+            };
+            this.snackBar.open(config);
```

Comment:
Probably not a part of the review, but have you noticed the amount of snackbar/snackbarconfigs and yet we never set them quite the same way? 
Assuming it's an upstream thing but wouldn't it be cleaner to make a method or service and have them all call it so it's the same code?

Reply:
The snackbar config is the way that upstream standardized the snackbar changes. 

Not sure what you mean by us setting them differently, I didn't see any major difference between us and upstream in terms of snackbars, outside of the localized messages.